src/AppBundle/Controller/Api/AccountController.php

```diff
@@ -82,4 +101,36 @@ public function getUserAction()
 
         return new JsonResponse($userDto);
     }
+
+
+    /**
+     * @param Request $request
+     *
+     * @Route(
+     *     "api/account/get_department",
+     *     methods={"GET"}
+     * )
+     *
+     * @return \Symfony\Component\HttpFoundation\Response
```

Comment:
Samme her som forrige punkt, Annotation er deprecated, og import kan brukes for `Response` slik at koden blir mer lesbar